Automattic/wp-calypso
# Emails: Disable Google Workspace with monthly billing and add Titan logo to In-Depth Comparison page

This pull request updates the `In-Depth Comparison` page to show a message and disable the `Select` button for Google Workspace when monthly billing is selected. It also adds a `Powered By Titan` logo for Professional Email:

Desktop | Mobile
------ | -----
![image](https://user-images.githubusercontent.com/594356/149987542-c847fa30-067e-47d9-a374-bfe41e8129e6.png)<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>| ![image](https://user-images.githubusercontent.com/594356/149987621-a46cfd6e-7d79-4b42-9883-5acac54031bd.png)



#### Testing instructions

Note you will have to append `?flags=emails/in-depth-comparison` to the url of the page (and reload it) if you're testing those changes on a live branch as this feature has not been enabled on this environment yet:

1. Run `git checkout update/in-depth-comparison-page-2` and start your server, or access a [live branch](https://github.com/Automattic/wp-calypso/pull/60141#issuecomment-1014741663)
2. Log into a WordPress.com account with a domain but no email
3. Open the [`Emails` page](http://calypso.localhost:3000/email)
4. Click the `Add Email` button of that domain to access the `Email Comparison` page
5. Click the `See how they compare` link to access the `In-Depth Comparison` page
6. Click the `Pay monthly`  toggle
7. Assert that `Only available with annual billing` is shown instead of a price for Google Workspace
8. Assert that `Select` button is disabled for Google Workspace
